shell/devspace_start.sh

```diff
@@ -74,13 +80,13 @@ This is how you can work with it:
 
 if [[ -z $DEV_CONTAINER_LOGFILE ]] || [[ $DEVENV_DEV_TERMINAL == "true" ]]; then
   echo -e "$BANNER"
-  bash
+  exec bash
```

Comment:
I forgot to mention this, using `exec` is better for ensuring signals are passed through to things appropriately. Just a minor cleanup.